params/hooks_libevm.go

```diff
@@ -98,8 +112,10 @@ func (r RulesExtra) ActivePrecompiles(existing []common.Address) []common.Addres
 // dynamic precompile activation registry.
 // These were only active historically and are not active in the current network.
```

Comment:
This comment is out of date.

Reply:
I take it this is no longer true? I'll remove the comment.

Reply:
Sorry, I think I misunderstood because of the use of a `DeprecatedContract` type that does its own deprecation. Pretend I didn't say anything 🤐